app/views/admin/finances/index.html.haml

```diff
@@ -12,3 +13,6 @@
 
 %h2= t('.bank_accounts')
 #bank_accounts_table= render 'bank_accounts'
+
+%h2= t('.bank_gateways')
+#bank_gateways_table= render 'bank_gateways'
```

Comment:
Perhaps we need to add `- if @bank_gateways.any?` (and on that note, `@bank_accounts.any?`) to make this prettier when not using this functionality. Or let it also depend on `use_iban`?

Reply:
i don't see the benefit of hiding it here, since it's a configuration page. we don't hide `bank_accounts` either

Reply:
My bad, I was thinking of finances overview, good to have here!